spline/validation.py

```diff
@@ -67,7 +67,8 @@ class Validator(object):
                         Optional('with'): And(len, [object]),
                         Optional('variable'):
                             And(Or(type(' '), type(u' ')), len, Regex(r'([a-zA-Z][_a-zA-Z]*)')),
-                        Optional('when', default=''): And(str, Condition.is_valid)
+                        Optional('when', default=''): And(str, Condition.is_valid),
+                        Optional('retries', default=1): int
```

Comment:
I wouldn't allow any number of retries ... 3 has been used often .... but 100 very probably does not make sense ... it also depends how long a task took .... for longer tasks I would rather choose a smaller number of retries ...

I working much on ci/cd and long durations are not accepted .... 
That's why I would - probably - limit the whole to 5 or 6 (maximum)

And also it might be reasonable to combine retry and wait 
as a pair of information ... like '3:4' ... 3 times and wait 4 seconds after each fail.

Reply:
I'm not super concerned about the maximum length it could take. All CI/CD systems have a maximum entire length of execution. In addition, I can actually think of reasons why to loop forever, oddly.

I do like the idea of being able to specify a wait interval, instead of exponential back-off. However, I am not sure of how the schema (validation) would look like. Would it be an `int` or a regexp matching it, then? On the other hand, it does convolute the expression, slightly. Maybe a different attribute could be used? Not sure.

Comments, suggestions, and ideas are always welcomed,
-Joe